com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -165,3 +165,8 @@ With:
 ```
 protected override void Execute(CustomPassContext ctx) { ... }
 ```
+
+## Density Volume Mask Texture
+
+Previously, Density Mask Textures needed the __Density Volume Texture Tool__ in the __Window > Rendering__ menu to be converted from a 2D flipbook format to the 3D format required by the Density Volume Component.
+With Unity 20202.2, this can be done directly through the __Texture Importer__. To see the detailed steps on how to use the importer, refer to the [Density Volume documentation](Density-Volume.md).
```

Comment:
From Unity 2020.2, you can now do this conversion directly through the __Texture Importer__. For information on how to use the importer, see to the [Density Volume documentation](Density-Volume.md).